routes/index.tsx

```diff
@@ -13,7 +13,11 @@ export default function LandingPage() {
 
   useEffect(() => {
     if (user) {
-      if (isUserReady(user)) {
+      if (
```

Comment:
remove `isUserReady` to avoid client import db deps

Reply:
makes sense